### PR TITLE
AA-2840 adding quick links to home page

### PR DESCRIPTION
**Pull request by ShiraziElham**
To Validate [AA-2840](https://applyboard.atlassian.net/browse/AA-2840) :
1- Verify the design!
2- Verify hyperlinks pointing to right end points.

### CODE REVIEW

File: templates/home_page.hbs [hunk 1]
```diff
@@ -159,7 +159,39 @@
...
   {{/if}}
-
+  <div class="clearfix container-inner">
+    <div class="knowledge-base">
+      <h4 class="section-header">
+        Quick Links
+      </h4>
+      <div class="row updated-category-list">
+        <ul class="column column--sm-3 section-list-item" id="school_id">
+          <p>Promotions &amp; Scholarships</p>
```
Comment (wesleymartin-ab) [inline templates/home_page.hbs:169]: IS it possible to hjave the first <p> child of the UL have a minimum height of 52? That will make sure that sure that the header rows are all the same height

**Review by wesleymartin-ab — COMMENTED** [review]
More details haha. The thing with making the headers the same height may not be doable, but give it a shot.

File: templates/home_page.hbs (hunk 1 above)
Comment (ShiraziElham) [reply templates/home_page.hbs:169]: It did not help!

**Review by wesleymartin-ab — APPROVED** [review]
I think its good now!

**Pull request merged**